src/content/docs/en/guides/deploy/clever-cloud.mdx

```diff
@@ -0,0 +1,121 @@
+
+3. Select a **Node.js** application, or a **Static** one.
+	4. Set up the minimal size for your instance and scalability options: Astro can be deployed in a **Nano** instance as minimal size. Instance sizes depends of your application specifications and dependencies. We suggest you start with the smallest instance and then adjust accordingly as you watch the metrics from the **Overview** page.
+5. Select a **region** where to deploy your instance.
+6. **Add-ons**: Unless you're using a database or Keycloak, you shouldn't need any add-on. See [Further Configuration](#further-configuration) if you do.
+7. Inject **environment variables**:
+
+    - For **Node.js**, no specific environment variable is needed to deploy Astro.
+
+    - For a **Static** application, add these variables:
+
+    ```shell
+    CC_POST_BUILD_HOOK="npm run build"
+    CC_PRE_BUILD_HOOK="npm install && npm run astro telemetry disable"
+    CC_WEBROOT="/dist"
+    ````
+8. **Deploy!** If you're deploying from **GitHub**, your deployment should start automatically. If you're using **Git**, copy the remote and push on **master** branch. 
+
+</Steps>
+
+:::tip[Other Branches]
+Deploy from other branches than `master` branch by using `git push clever <branch>:master`. For example, if you don't want to rename your local `main` branch, use `git push clever main:master`.
+:::
+
+## Further configuration
+
+This section explores further configuration for your Astro application on Clever Cloud.
+
+### Scalability
+
+Enable auto-scalability in **Scalability** option, to automatically allocate ressources to the server according to its consumption. This is especially recommended if your Astro app uses Server-Side Rendering (SSR).
+
+#### Dedicated build instance
+
+Aditionnally, enabling a dedicated build instance is available from the **Information** otpion in your application menu. Once the build succeeds, your application runs in the minimal instance size you've set.
+
+Check your metrics in the application **Overview** during build, and activate if you see consumption pics during the build phase.  
+
+Click the Grafana icon on the metrics overview to access the application dedicated Grafana dashboard for more detailed metrics.
+
+### Add-ons: Databases, Keycloak, Analytics
+
+Clever Cloud provides different add-ons you could find useful to connect to your Astro application, such as:
+
+- **Databases**: PostrgreSQL, MySQL, MongoDB, Redis...
+- **Keycloak**: If your application uses [Keycloak auth provider from Auth.js](https://authjs.dev/getting-started/authentication/oauth), Clever Cloud provides a Keykloak add-on.
+- **Matomo**: To monitor your application web analytics as an alternative to Google Analytics.
+
+To connect a service to your Astro application, create an add-on and connect it to your application directly from the Console. See all available add-ons on [Clever Cloud documentation](https://developers.clever-cloud.com/doc/addons/).
```

Comment:
This contains helpful info, however deployment guides should be succinct, and these sit outside the goal of the recipe.
```suggestion
```